### PR TITLE
Added My First Keymap

### PR DESCRIPTION
**Pull request by Magascorada**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/kyria/keymaps/MagasCorada [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (Spaceman) [inline keyboards/kyria/keymaps/MagasCorada:1]: This file should be deleted

Comment (Spaceman) [reply keyboards/kyria/keymaps/MagasCorada:1]: Other than that this PR looks good to me

**Review by Spaceman — CHANGES_REQUESTED** [review]

**Comment by Magascorada** [comment]
I don't know how to delete the file. I'm sorry.

**Comment by JunoNgx** [comment]
You created an empty file named `MagasCorada` (without an extension) in the `kyria` folder. This is a file, not a folder and it doesn't do anything. The quickest way is to delete it with the github web GUI.

Else you can just delete it locally from your repository clone and push to the change to remote.

File: keyboards/kyria/keymaps/Magascorada/keymap.c [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+enum layers {
+    _QWERTY = 0,
```
Comment (drashna) [inline keyboards/kyria/keymaps/Magascorada/keymap.c:19]: The zero isn't needed here, as enums start at 0, and increment up from there.
```suggestion
    _QWERTY,
```

**Review by drashna — CHANGES_REQUESTED** [review]
Also, if you could rename all of the files so that they're all lower case? 

You should be able to do that in the webUI for github, if needed.

**Comment by drashna** [comment]
If you're still having issues with removing the file, you can do this through github's UI.

Via command line, it would be:
```
git rm keyboards/kyria/keymaps/MagasCorada
```

And then `git commit`.

**Pull request closed without merging**